<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>838</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>598</width>
    <height>838</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>598</width>
    <height>838</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animesonline.club Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_fundo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>821</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>600</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>581</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>581</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="3">
      <widget class="QLineEdit" name="ed_fim">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Epis√≥dios Encontrados</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Digite a sequencia que deseja baixar: de</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ed_inicio">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="bt_baixar_eps">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Baixar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QPlainTextEdit" name="lista_epsodios">
       <property name="minimumSize">
        <size>
         <width>570</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>590</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>582</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="ed_nome">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bt_pesquisar">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Pesquisar Anime</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QListWidget" name="lista_animes">
        <property name="minimumSize">
         <size>
          <width>580</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>580</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Pesquisar Animes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="bt_abre_pasta">
     <property name="geometry">
      <rect>
       <x>523</x>
       <y>9</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Downloads</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ed_nome</tabstop>
  <tabstop>bt_pesquisar</tabstop>
  <tabstop>lista_animes</tabstop>
  <tabstop>lista_epsodios</tabstop>
  <tabstop>ed_inicio</tabstop>
  <tabstop>ed_fim</tabstop>
  <tabstop>bt_baixar_eps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
